t/t2230-job-info-list.t

```diff
@@ -209,42 +210,48 @@ test_expect_success HAVE_JQ 'flux job list inactive jobs in completed order' '
 
 test_expect_success HAVE_JQ 'flux job list inactive jobs with correct state' '
         for count in `seq 1 6`; do \
-            echo "64" >> list_state_I.exp; \
+            echo "INACTIVE" >> list_state_I.exp; \
         done &&
-        flux job list -s inactive | jq .state > list_state_I.out &&
+        flux job list -s inactive | jq .state | ${JOB_CONV} statetostr > list_state_I.out &&
         test_cmp list_state_I.out list_state_I.exp
 '
 
 test_expect_success HAVE_JQ 'flux job list inactive jobs results are correct' '
-        flux job list -s inactive | jq .result > list_result_I.out &&
-        echo "4" >> list_result_I.exp &&
-        echo "2" >> list_result_I.exp &&
+        flux job list -s inactive | jq .result | ${JOB_CONV} resulttostr > list_result_I.out &&
+        echo "CANCELLED" >> list_result_I.exp &&
```

Comment:
Shall we standardize on the American spelling of "canceled' (one el?)  We're using both.

https://www.merriam-webster.com/words-at-play/canceled-or-cancelled

Reply:
I too prefer the American spelling (didn't realize it was American v British thing), but I think we're already using the double-el version more often (and it is part of the API in `flux_job_result_t` enum). I'd be fine with making the switch now if someone wants to do the work of the update.

Reply:
Maybe we can support both `FLUX_JOB_RESULT_CANCELED` and `FLUX_JOB_RESULT_CANCELLED`, and also either `CANCELLED` or `CANCELED` for the resulting string-to-state/state-to-string operations...

Reply:
that sounds good.  I'll open up a new issue, take care of that in a later PR.

Reply:
> Maybe we can support both FLUX_JOB_RESULT_CANCELED and FLUX_JOB_RESULT_CANCELLED, and also either CANCELLED or CANCELED for the resulting string-to-state/state-to-string operations...

I think it sets a bad precedent.  Someday we might need to support COLOUR, DEFENCE, and TRAVELLED as well.  I think its better to stick with one for better or for worse.

Reply:
Good point! Though in this case it is for backwards compatibility of existing users. *Hopefully* nobody is using these interfaces yet though...